app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java

```diff
@@ -677,8 +677,18 @@ public void setTitle(String title) {
     }
 
     private String getRunTime() {
-        Long runtime = Utils.getSafeValue(mBaseItem.getRunTimeTicks(), mBaseItem.getRunTimeTicks());
-        return runtime != null && runtime > 0 ? String.format("%d%s", (int) Math.ceil((double) runtime / 600000000), getString(R.string.lbl_min)) : "";
+        Long runtime_ticks = Utils.getSafeValue(mBaseItem.getRunTimeTicks(), mBaseItem.getRunTimeTicks());
```

Comment:
Could probbaly use `TimeUtils.formatMillis` here, which is used in other places.

Reply:
Yeah, not really. It displays time in the format `10:05:55` instead of the format the jellyfin web client uses.  I would need to heavily modify this function to be able to use it.